### PR TITLE
change about ex3_17.cpp and ex3_20.cpp

### PR DESCRIPTION
**Pull request by mudongliang**
1.change the output of ex3_17.cpp,because the demand of question is that one result occpies one line. 2.That put the two solutions in one file-ex3_20.cpp will confuse the logic,so I separate it into two files and revise them!

### CODE REVIEW

File: ch03/ex3_17.cpp [hunk 1]
```diff
@@ -29,12 +29,7 @@ int main()
...
 
   for (decltype(vec.size()) i=0; i != vec.size(); ++i)
-  {
-    if (i != 0 && i%8 == 0) cout << endl;
-    cout << vec[i] << " ";
-  }
-  cout << endl;
-
+    cout << vec[i] << endl;
   return 0;
```
Comment (pezy) [inline ch03/ex3_17.cpp]: Why ? Review the question:
> Print the transformed elements, **eight words** to a line.

**Comment by pezy** [comment]
> put the two solutions in one file-ex3_20.cpp will confuse the logic

:no_good: , the two solutions doesn't contradict.

File: ch03/ex3_17.cpp (hunk 1 above)
Comment (mudongliang) [inline ch03/ex3_17.cpp]: 我看的是中文版的，这上面是：“输出改变后的结果，每个词占一行”，我不知道两者翻译是不是有什么问题？

Comment (pezy) [inline ch03/ex3_17.cpp]: 以英文版为主.

**Pull request closed without merging**